app/org/sagebionetworks/bridge/dynamodb/StringIntegerMapMarshaller.java

```diff
@@ -0,0 +1,12 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import java.util.Map;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.JsonMarshaller;
+
+/**
+ * Marshalls Map<String, Integer> values JSON when persisting using DynamoDB.
+ */
+public class StringIntegerMapMarshaller extends JsonMarshaller<Map<String,Integer>> {
+
+}
```

Comment:
What was your experience in needing this, because there's no implementation here. What did DynamoDB do without this?